ci/openshift-ci/cluster/deployments/workaround-9206-crio.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 10-workaround-9206-crio
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+              source: data:text/plain;charset=utf-8;base64,W2NyaW9dCnN0b3JhZ2Vfb3B0aW9uID0gWwoJIm92ZXJsYXkuc2tpcF9tb3VudF9ob21lPXRydWUiLApdCg==
```

Comment:
Did you consider to have the config file versioned in plain-text then encoded to base64 on runtime?

Like it does to apply SELinux config -> https://github.com/kata-containers/kata-containers/blob/main/ci/openshift-ci/cluster/deployments/machineconfig_selinux.yaml.in#L20C62-L20C81

The advantage of plain-text is that it's reviewable more easily :D

Reply:
I can do that but it just seemed better, condensed and hopefully nobody will have to modify it ever :-)

Reply:
ok, your call :)